parquet-protobuf/src/main/java/org/apache/parquet/proto/ProtoMessageConverter.java

```diff
@@ -86,32 +89,71 @@ class ProtoMessageConverter extends GroupConverter {
     this.conf = conf;
     this.parent = pvc;
     this.extraMetadata = extraMetadata;
-    int parquetFieldIndex = 1;
+    boolean ignoreUnknownFields = conf.getBoolean("IGNORE_UNKNOWN_FIELDS", false);
```

Comment:
Seems line 92-94 can be moved under line 98?

Reply:
I think the whole check can for pvv==null can be moved up. I have done that. Please let me know if you disagree